src/search/components/SearchPage.js

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react';
```

Comment:
I did feel like I was being forced to write it this way a little.

The docs suggest that you don't have any redux in your `components` (presentational components) and the `containers` (container components) are automatically generated to pass the redux data into your `components`. This seems like a reasonable approach so I went with it.

> Now it’s time to hook up those presentational components to Redux by creating some containers. Technically, a container component is just a React component that uses `store.subscribe()` to read a part of the Redux state tree and supply props to a presentational component it renders. You could write a container component by hand but React Redux includes many useful optimizations so we suggest to generate container components with `connect()` function from the React Redux library.

From [Usage with React](http://redux.js.org/docs/basics/UsageWithReact.html) (doesn't support linking very well, because JavaScript)